variants/backend-base/config/initializers/version.rb

```diff
@@ -1,12 +1,6 @@
 Rails.application.config.version = begin
-  `git describe --always --tag 2> /dev/null`.chomp
```

Comment:
btw fwiw I'm pretty sure this wouldn't have worked even with Capistrano, because it doesn't run the application in an initialized `git` repo:

![Uploading image.png…]()